connman/client/commands.c

```diff
@@ -318,6 +319,18 @@ static int peers_list(DBusMessageIter *iter,
 	return 0;
 }
 
+static int tethering_clients_list(DBusMessageIter *iter,
+					const char *error, void *user_data)
```

Comment:
Unused variable user_data... Maybe same at upstream. Could cause issues with compiler.

Reply:
I think this is left there for future use. Need to check what is in the upstream now. But anyways, this is the `connmanctl` binary for accessing options, services etc. via cli.

Reply:
Ah, no. This is what the function prototype has to have. It is a callback to D-Bus call. Should not be touched.